corehq/apps/smsbillables/management/commands/bootstrap_gateway_fees.py

```diff
@@ -37,21 +39,35 @@ def _bootstrap_gateway(apps, backend):
         SmsGatewayFee.create_new(
             backend.get_api_id(),
             direction,
-            None,
+            amount,
             fee_class=sms_gateway_fee_class,
             criteria_class=sms_gateway_fee_criteria_class,
             currency=default_currency,
+            country_code=country_code,
```

Comment:
I assume that `country_code=None` is the same as leaving off `country_code` in this method call (i.e. `None` is the default value for `country_code`)? If so then this looks good

Reply:
Yeah here's the definition for that `SmsGatewayFee` class method:
https://github.com/dimagi/commcare-hq/blob/015db108d034861004a5918c575b4d952020a6c6/corehq/apps/smsbillables/models.py#L125-L128